payments-core/res/values/totranslate.xml

```diff
@@ -4,5 +4,16 @@
     Items here need to be entered in our translation tool
     -->
     <!-- Affirm buy now, pay later copy -->
-    <string name="affirm_buy_now_pay_later">Buy now, pay later with Affirm.</string>
+    <string name="affirm_buy_now_pay_later">Buy now, pay later with</string>
+    <!-- An error message displayed when the customer's BSB number is incomplete. -->
+    <string name="bsb_incomplete">The BSB number you entered is incomplete.</string>
+    <!-- An error message displayed when the customer's BSB number is invalid. -->
+    <string name="bsb_invalid">The BSB number you entered is invalid.</string>
```

Comment:
Let's reuse the strings for the bsb_widget already in payments-core.  See becs_widget_bsb_incomplete, becs_widget_bsb_invalid, becs_widget_account_number_incomplete, etc.